setup.py

```diff
@@ -82,6 +86,7 @@
     url='https://github.com/python-cmd2/cmd2',
     license='MIT',
     platforms=['any'],
+    package_data=PACKAGE_DATA,
```

Comment:
What effect does adding this have?

Reply:
This causes py.typed to be included in the wheel.